ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -1021,11 +1021,46 @@ let start () =
     Fdecl.get server
   in
   let state = Server.state server in
+  let log_errors what f =
+    let+ res =
+      Fiber.map_reduce_errors
+        (module Monoid.Unit)
+        f
+        ~on_error:(fun exn ->
+          Format.eprintf "%s: %a@." what Exn_with_backtrace.pp_uncaught exn;
+          Fiber.return ())
+    in
+    match res with
+    | Ok () -> ()
+    | Error () -> ()
```

Comment:
this also looks very verbose for very innocuous thing, IMHO. I would create a util function in `Import.Fiber`